src/stores/chart-store.ts

```diff
@@ -66,14 +66,12 @@ export default class ChartStore {
 
     updateSymbol = () => {
         const workspace = window.Blockly.derivWorkspace;
-        const market_block = workspace.getAllBlocks().find((block: window.Blockly.Block) => {
+        const market_block = workspace?.getAllBlocks().find((block: window.Blockly.Block) => {
             return block.type === 'trade_definition_market';
         });
 
-        if (market_block && market_block !== 'na') {
-            const symbol = market_block.getFieldValue('SYMBOL_LIST');
-            this.symbol = symbol;
-        }
+        const symbol = market_block?.getFieldValue('SYMBOL_LIST') || '1HZ10V';
```

Comment:
```suggestion
        const symbol = market_block?.getFieldValue('SYMBOL_LIST') ?? '1HZ10V';
```

Also, can we take the value from the active symbol list instead of directly putting it here? There could be a scenario where this `1HZ10V` may not be available.